src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+public class Ui {
+
+    public Ui() {
+        System.out.println("Hello!!! I'm jBot\nWhat can I do for you? :3");
+
+    }
+
+    public void showLoadingError() {
```

Comment:
For this kind of message, you may want to define them as constants so that it might be easier when you want to modify it or reuse it somewhere else.